app/res/drawable/ic_cc_update.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="25dp" android:viewportHeight="232.5"
```

Comment:
any reason we can't use the existing `.svg` file here?

Reply:
I converted it to vector drawable because their rendering tends to be faster and compatibility with older android versions is higher. Is there any particular advantage with using `svg` files?

Reply:
no was just curious why we need to create another file.